src/plugins/discover/public/application/_mixins.scss

```diff
@@ -1,27 +0,0 @@
-/**
-*  Style ES document _source in table view <dt>key:<dt><dd>value</dd>
-*  Use alpha so this will stand out against non-white backgrounds, e.g. the highlighted
-*  row in the Context Log.
-*/
-@mixin dscDocSourceStyle {
```

Comment:
This file was appropriate. The mixins need to live in their own file in the plugin and loaded at the top level even if they're only used in a single file downstream.

Reply:
thx, will move it to a _mixins.scss on top level of Discover

Reply:
it was just used 2 times in Discover, changed the Styles a but now there's longer a mixin necessary